routes/web.php

```diff
@@ -24,6 +27,18 @@
 Route::get('/users_complete', 'UsersController@usersLike');
 Route::get('/users_parcial', 'UsersController@usersOnlyLike');
 
+Route::get('permission',function () {
+    return view('permissions', ['users' => User::with("permission"
+)->get()]);
+});
+
+Route::put('permission/{id}',function (Request $request, $id) {
+	$change_permission = App\User::find($id);
+	$change_permission->idUserType = $request["permission"];
+	$change_permission->save();
+	return Redirect::to("permission");
```

Comment:
we use return redirect()->route("name") in the rest of controllers. Can you change it to be consistent?